.gitignore

```diff
@@ -30,3 +30,4 @@ nbproject/
 # Eclipse
 .project
 .settings/
+package.json
```

Comment:
This is incorrect. package.json is in git

Reply:
Had to exclude it because my git desktop refused to upload the changes because of validation of existing files failing.

So ignored the file to on theory prevent it being uploaded as I made a local mod to it